tests/routes.users.js

```diff
@@ -33,35 +35,38 @@ describe('POST /api/users', () => {
                 age: 30
             }
         };
+        const userArr = await User.findAll({ raw: true });
+        const userBeforeLength = userArr.length;
 
         const res = await supertest(app)
             .post('/api/users')
             .send(testUser)
             .expect(httpStatus.OK);
 
-        const userById = await User.findByPk(res.body.user.id);
+        const userArrAfter = await User.findAll({ raw: true });
+        const userAfterLength = userArrAfter.length;
 
-        assert.deepStrictEqual(testUser.user.firstName, userById.firstName, 'create correct user');
+        assert.strictEqual(userBeforeLength + 1, userAfterLength, 'create correct user');
     });
 });
 
 describe('GET /api/users/:id', () => {
     it('should return single user', async () => {
-        const testUser = {
+        const newUser = {
             user: {
                 firstName: 'testName',
                 age: '25'
             }
         };
-        const newUsers = await User.create(testUser.user);
+        const newUsers = await User.create(newUser.user);
```

Comment:
1)`const newUsers` ? newUser is should be
2) Why do you do that `const userById = await User.findByPk(res.body.user.id);` You already have this user in `newUser` variable